Homework-16

Gtandrea1993 opened the pull request:
Done

Gtandrea1993 committed:
Homework-16

Done

Berichicko commented:
Please, add Homework16 in TestNG.xml to run tests

For example

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/ff353b30-fd7b-4003-9c00-9852d1391375)

Berichicko commented:
Hello,

I have checked your homework 16, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![804deb64dc406a20a75da05d91a1a69e](https://user-images.githubusercontent.com/104761482/235961479-9afb2841-b657-46c9-99f8-2402f933b843.jpg)

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/62f19952-6874-4331-a992-381ff0928513)

Gtandrea1993 committed:
Homework-17

search Song
Click View All
Select First Song
add To
choose PlayList
notification
